packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -20,7 +20,11 @@ import 'view.dart';
 export 'package:flutter/gestures.dart' show HitTestResult;
 
 /// The glue between the render tree and the Flutter engine.
-abstract class RendererBinding extends BindingBase implements SchedulerBinding, ServicesBinding, HitTestable {
+abstract class RendererBinding extends BindingBase with SchedulerBinding, ServicesBinding, HitTestable {
```

Comment:
Yes, DFE expects to find a concrete method in the super class of the mixin with a super call. Remember that the RendererBinding is also a regular class, and when we're compiling it, we may not know yet that someone intends to use it as a mixin.

My understanding is that the language team is considering this aspect of mixins and allow you to declare intent.